downgrade

```diff
@@ -448,12 +425,41 @@ parse_options() {
 }
 
 cli() {
-  parse_options "$@"
+  declare -a system_args
```

Comment:
Does `declare` within a function imply `local`? I have no idea. I would usually do `local system_args=()`.

I also wonder if `configuration_args` / `conf_args` is a better name. I'm not sure there's much meaning in "system" yet.

Reply:
I can't recall where I read this, but `declare` does imply locality.

But I agree with you, using the `local` syntax is more consistent with the coding style we have in `downgrade`. Will also change the variable name to `conf_args` :+1: